web/app/AttributeFormatter.js

```diff
@@ -78,6 +82,8 @@ Ext.define('Traccar.AttributeFormatter', {
             return this.hoursFormatter;
         } else if (key === 'duration') {
             return this.durationFormatter;
+        } else if (key === 'deviceName') {
```

Comment:
Key should be `deviceId`. That's the naming convention to have the same name as the model key.